h/static/styles/annotation.scss

```diff
@@ -10,6 +10,10 @@
   background-color: $white;
 }
 
+.annotation-quote-list--orphan {
+  text-decoration: line-through;
```

Comment:
Looking at [this screenshot](https://trello-attachments.s3.amazonaws.com/574dae8f7cd863375a5643c8/2180x1100/e62fb94d49354e653faa88e9d289b169/Unanchored_tab.png) from the Trello card, it looks as though the strike-through line for quotes is meant to be a lighter shade of grey than what I see here.